README.md

```diff
@@ -28,8 +28,8 @@ Please go through the [Contribution Guidelines](CONTRIBUTING.md) before going fo
 **Note:** You might want to use the docker instructions if you do not want to setup your own environment.
 
 * Install ruby using RVM, use ruby-2.5.1
-* Install Dependencies: `bundle install --with mysql` (If you wanto use pg use `bundle install --with pg`)
-* Configure your DB in config/database.yml, copy config/database.yml.example
+* Install Dependencies: `bundle install --with pg` 
```

Comment:
I guess `--with pg` will not be required now as it will the be the default db for all the environments(test, prod and dev).